demo/App.js

```diff
@@ -1,12 +1,19 @@
 import React from "react";
 import { StyleSheet, Text, View } from "react-native";
 import { AppLoading, Asset, Font } from "expo";
+//import { Account } from "tasit-sdk";
 
 export default class App extends React.Component {
   state = {
     isLoadingComplete: false
   };
 
+  componentDidMount() {
+    // Account.create().then(w => {
```

Comment:
@marcelomorgado Even though this isn't being used since it's commented out and the dep isn't live on npm yet, I'd  still appreciate it if we could swap it to async/await style. Thanks!

Reply:
OK